tests/System.Data.SQLite.Tests/System.Data.SQLite.Tests.csproj

```diff
@@ -33,6 +33,11 @@
     <ExtrasBuildEachRuntimeIdentifier>true</ExtrasBuildEachRuntimeIdentifier>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Platform)' == 'everything' and '$(TargetFramework)' == 'net5.0' ">
```

Comment:
Could use `or` with `netcoreapp3.1`. Might be simpler if you eventually add `net6.0`. But this is fine too.